modules/renter/upload.go

```diff
@@ -140,10 +141,11 @@ func (r *Renter) Upload(up modules.FileUploadParams) error {
 	// number of pieces plus some buffer before we decide that an upload is
 	// okay.
 	if len(r.hostDB.ActiveHosts()) < 1 {
-		return errors.New("not enough hosts on the network to upload a file :( - maybe you need to upgrade your software")
+		return errors.New("not enough hosts on the network to upload a file")
```

Comment:
If there aren't enough hosts it's probably because they need to update their client.

I feel like they should be told that.